packages/aws-serverless/package.json

```diff
@@ -40,6 +40,16 @@
       "import": {
         "default": "./build/loader-hook.mjs"
       }
+    },
+    "./awslambda-auto": {
+      "import": {
+        "default": "./build/npm/cjs/awslambda-auto.js"
+      }
+    },
```

Comment:
`import` is used for ESM but the file is CJS. I don't think this will work (or is necessary) and we can remove this block